BridgeAppSDK/SBANewsFeedParser.h

```diff
@@ -1,8 +1,4 @@
-//
-//  SBATaskViewController.h
-//  BridgeAppSDK
-//
-//  Copyright © 2016 Sage Bionetworks. All rights reserved.
+// Copyright (c) 2015, Apple Inc. All rights reserved.
```

Comment:
probably Git is being weird thinking these were ever the same file, but does copyright need updating here or was the file copied unchanged?